src/libraries/Microsoft.PowerFx.Core/Entities/External/TableDelegationInfo.cs

```diff
@@ -52,6 +52,10 @@ public abstract class TableDelegationInfo
         // Dataset name
         public string DatasetName { get; init; }
 
+        // Supports primary key names (multiple when composed key)
+        // This array is ordered
+        public string[] PrimaryKeyNames { get; init; }
```

Comment:
>string[] [](http://example.com/codeflow?start=15&length=8)

nit - arrays are mutable. 
Can we expose this as something immutable  like IReadOnlyCollection<string> ?